consensus/merge/src/test/java/org/hyperledger/besu/consensus/merge/blockcreation/MergeCoordinatorTest.java

```diff
@@ -317,6 +317,26 @@ public void exceptionDuringBuildingBlockShouldNotBeInvalid()
     verify(willThrow, never()).addBadBlock(any(), any());
   }
 
+  @Test
+  public void shouldNotRecordProposedBadBlockToBadBlockManager()
```

Comment:
Is there a test already to ensure that an invalid block we received _does_ get added to the bad blocks?